src/test/java/de/qabel/desktop/ui/remotefs/RemoteFSControllerTest.java

```diff
@@ -197,7 +200,7 @@ public void testUploadedDirectoryInChildNode() throws QblStorageException, IOExc
     @Test
     public void testCreateFolder() throws QblStorageException {
 
-        File dir = new File("tmp/testFolder");
+        File dir = new File(TMP_DIR +"/" + TEST_FOLDER);
```

Comment:
pretty redundant. If all tests use the same dir, define it in a setUp method or extract a field